### DISCUSSION TITLE
Field Guide

### DISCUSSION THREAD

**Discussion by bigolboyyo** [climate-tech-handbook/climate-tech-handbook#4] [General]
https://github.com/climate-tech-handbook/data-magic/pull/4

Making this discussion to talk about our guide. 

What's our stack? What technologies will be incorporated? 
What instructions, tutorials do we need to provide? 
What is our criteria for submitting pull requests?

And much more!

**Comment by bigolboyyo** [comment]
Currently I have pushed the initial instructions and guide to data magic: https://github.com/climate-tech-handbook/data-magic

We will want to consolidate everything into the main handbook repo though: 
https://github.com/climate-tech-handbook/climate-tech-handbook

**Comment by bigolboyyo** [comment]
Noticing discrepancies in formatting. 

TODO: We should have the specific goal of figuring out the sector/solution/resources templating and file structure

An example of our output in markdown
```
# Nuclear Power

## Progress Made

Nuclear power is a clean and efficient way to produce electricity. It does not produce greenhouse gas emissions, so it is an important technology for reducing climate change. There have been many breakthroughs in nuclear power technology in recent years, including new designs for nuclear reactors that are safer and more efficient, and new methods for recycling nuclear fuel. These advances have made nuclear power more attractive as a clean energy option and have contributed to the reduction of greenhouse gas emissions.

## Lessons Learned
...
```

Where as in the original theme md i'm seeing something like
```
---
Title:
Description: 
---

# Biochar Production

!!! info "[Help us](../../contribute) track this Solution"

<div class="grid cards" markdown>

-   :material-check-bold:{ .lg .middle } __##__ Gigatons saved from [carbon budget](glossary/#carbon-budget)

</div>

## Solution Overview

_insert description_


## Progress Made
```

This would be a great task for someone who is still just learning HTML/CSS as you can do basic web design on markdown.


I know @futuresoup mentioned wanting the simplicity of a Solutions and Resources page as the only two pages (outside of home/about) 


I mentioned the file structure as well: I still need to deep dive MaterialMK to fully understand custom scripting and theming.

But it seems pretty straightforward, so if there is a specific structure we need to maintain there then that's understood. If we can organize it a little better than that would be great (insert office space meme here). This will help immensely when parsing and uploading our data to the frontend.

An example of what I'm talking about would be something like this.

![image](https://user-images.githubusercontent.com/95878821/235540653-d85a0e92-6385-4f0f-8d73-c84a20f953b0.png)

We have all of our sectors listed but if we look at our directory.

`docs/solutions/sectors`

In this instance notice that electricity isn't in there, it's actually further up:

`docs/sector-electricity.md`

This can obviously cause lots of page mismatches, client side routing issues, etc.

Let me know if I'm mistaken in any of this and what your findings are!

**Comment by futuresoup** [comment]
Closing this. It will be possible once we create a way to batch update pages [via this Issue](https://github.com/climate-tech-handbook/data-magic/issues/14).
